go.mod

```
@@ -3,17 +3,94 @@ module github.com/red-hat-storage/odf-cli
 go 1.21.0
```

required for compilation
```suggestion
go 1.21
```